src/content/app/species-selector/state/species-selector-search-slice/speciesSelectorSearchSlice.ts

```diff
@@ -25,21 +25,17 @@ import { saveMultipleSelectedSpecies } from 'src/content/app/species-selector/se
 import { getCommittedSpecies } from 'src/content/app/species-selector/state/species-selector-general-slice/speciesSelectorGeneralSelectors';
 
 import { updateCommittedSpecies } from 'src/content/app/species-selector/state/species-selector-general-slice/speciesSelectorGeneralSlice';
-import { setModalView } from 'src/content/app/species-selector/state/species-selector-ui-slice/speciesSelectorUISlice';
 
 import type { RootState } from 'src/store';
-import type { PopularSpecies } from 'src/content/app/species-selector/types/popularSpecies';
 import type { SpeciesSearchMatch } from 'src/content/app/species-selector/types/speciesSearchMatch';
 import type { CommittedItem } from 'src/content/app/species-selector/types/committedItem';
 
 type State = {
   query: string;
-  popularSpecies: PopularSpecies | null;
 };
 
 const initialState: State = {
-  query: '',
-  popularSpecies: null
+  query: ''
```

Comment:
I wonder if we still need to keep track of the query in redux. I can't think of what it might be needed for now.